src/dreyfus_index.erl

```diff
@@ -118,6 +119,17 @@ init({DbName, Index}) ->
             proc_lib:init_ack(Error)
     end.
 
+maybe_create_local_purge_doc(Db, IndexPid, Index) ->
+    DocId = dreyfus_util:get_local_purge_doc_id(Index#index.sig),
+    {ok, PurgeSeq} = clouseau_rpc:get_purge_seq(IndexPid),
```

Comment:
Hi Mayya. 
clouseau will return the default value 0 if it has not yet store a purge_seq.
And if the local purge doc is not exist here, it means purge_index is not called. The purge_seq should be 0 here.
And I am not sure the means "get purge_seq from a database". The purge seq is not kept in database now.
And the line #124 should be move to line #126, thank you, I will move this line.